cmd/bridge/main.go

```diff
@@ -561,15 +558,6 @@ func main() {
 		os.Exit(1)
 	}
 
-	// Setting the --k8s-auth-bearer-token flag will override both the authenticator
-	// and the internal proxy user
-	if len(*fK8sAuthBearerToken) > 0 {
-		srv.Authenticator = static.NewStaticAuthenticator(auth.User{
-			Token: *fK8sAuthBearerToken,
-		})
-		srv.InternalProxiedK8SClientConfig.BearerToken = *fK8sAuthBearerToken
```

Comment:
you'll want to keep this line somewhere

Reply:
See https://github.com/openshift/console/pull/13713/files#diff-ae1fd264e062e64b8683f373f7ef6d59c5903f5471fa6dad594f9d505ac43681R223-R225